### PR TITLE
Fix bug 1587924 - Remove Rank column from the Contributors table

### PR DESCRIPTION
**Pull request by Karskaya**
Remove rank from contributors.css and contributor_list.html [1587924](https://bugzilla.mozilla.org/show_bug.cgi?id=1587924)

### CODE REVIEW

**Comment by Karskaya** [comment]
Rank column removed from Contributors table.

File: pontoon/contributors/static/css/contributors.css [hunk 1]
```diff
@@ -23,16 +23,6 @@ body.top-contributors #heading .legend li .status.fa {
...
-
-td.rank {
-  color: #EBEBEB;
-  font-size: 36px;
-  width: 41px;
-}
-
 .contributor {
   display: inline-block;
   position: relative;
```
Comment (mathjazz) [inline pontoon/contributors/static/css/contributors.css]: Please set the `.contributor` with to `580px`.

Comment (Karskaya) [reply pontoon/contributors/static/css/contributors.css]: Done!

Comment (mathjazz) [reply pontoon/contributors/static/css/contributors.css]: Well done! I'm sorry, but we should also increase the `max-width` of `.contributor p` to `520px`.

**Comment by mathjazz** [comment]
That doesn't look like it's fixed?
https://github.com/mozilla/pontoon/pull/1418#discussion_r334670342

File: pontoon/contributors/static/css/contributors.css (hunk 1 above)
Comment (Karskaya) [reply pontoon/contributors/static/css/contributors.css]: > That doesn't look like it's fixed?
> [#1418 (comment)](https://github.com/mozilla/pontoon/pull/1418#discussion_r334670342)

Just saw it. Done!

**Pull request merged**